source/AMDSClient.h

```diff
@@ -31,6 +32,7 @@ private slots:
 	void onSocketDataReady(AMDSClientTCPSocket* socket, AMDSClientRequest *clientRequest);
 
 	void onSocketError(AMDSClientTCPSocket *clientTCPSocket, QAbstractSocket::SocketError socketError);
+	void onRequestTypeChanged(QString);
```

Comment:
Thx @dretrex the AMClient.h is removed and redefined in AMClientUi.h in a later PR. I applied these changes in there